mod_conf_sql.c

```diff
@@ -78,9 +75,9 @@ struct {
 #define SQLCONF_DEFAULT_CONF_ID_NAME    "conf_id"
 #define SQLCONF_DEFAULT_CTXT_ID_NAME    "ctxt_id"
 #define SQLCONF_DEFAULT_ID_NAME         "id"
-#define SQLCONF_DEFAULT_KEY_NAME       	"key"
+#define SQLCONF_DEFAULT_KEY_NAME        "type"
 #define SQLCONF_DEFAULT_PARENT_ID_NAME  "parent_id"
-#define SQLCONF_DEFAULT_VALUE_NAME      "value"
+#define SQLCONF_DEFAULT_VALUE_NAME      "info"
```

Comment:
because the documentation use this name... i changed that for concistency ...
but i can change this in reverse... and change documentation to be concistante... if you prefer...